test/functional/p2p_addr_relay.py

```diff
@@ -16,22 +16,11 @@
 from test_framework.util import (
     assert_equal,
 )
-import time
-
-ADDRS = []
```

Comment:
Can `ADDRS` remain outside?

In the last backport that modified `p2p_addr_relay.py` (bitcoin#19272), it has [remained outside](https://github.com/jonatack/bitcoin/blob/56010f92564a94b0ca6c008c0e6f74a19fad4a2a/test/functional/p2p_addr_relay.py#L24-L31). Scope pollution shouldn't be much of a concern since tests don't include each other.

I mention this because a similar change was done for `p2p_addrv2_relay.py` (see [`develop`](https://github.com/dashpay/dash/blob/85caf8aa34a9726ac83574b38df2fe47af9ae6de/test/functional/p2p_addrv2_relay.py#L42-L49) vs [bitcoin#19954](https://github.com/vasild/bitcoin/blob/dcf0cb477699d11afd0ff37c8bfb2b1b4f7f1ee5/test/functional/p2p_addrv2_relay.py#L21-L28), the backport that introduced this test) and I had to revert it in a WIP PR to make way for bitcoin#22211 because `ADDRS` was being accessed by both `AddrReceiver` and `AddrTest` in the [changes introduced](https://github.com/bitcoin/bitcoin/pull/22211/commits/7593b06bd1262f438bf34769ecc00e9c22328e56#diff-d3d7b1bb23f25a96c9c7444a79159ad1799895565f99efebf1618e41e886bd53R21-R46).

Also, `ADDRS` in `p2p_addr_relay.py` is anyways done away with in bitcoin#21707 ([diff](https://github.com/bitcoin/bitcoin/pull/21707/files#diff-5a9f30817894348260893f195cea07da88271b5e01b1531f0cd839ee85898592L22-L32)) so might be worthwhile to keep it in line with upstream to make the changes easier to follow when it is eventually done away with.